enterprise/server/util/firecracker_util/firecracker_util.go

```diff
@@ -0,0 +1,31 @@
+package firecracker_util
```

Comment:
nit: we haven't been consistent with this but the go style guide recommends against underscores in package names